src/albert/collections/custom_templates.py

```diff
@@ -87,6 +87,30 @@ def search(
             ],
         )
 
+    def create(self, *, custom_template: CustomTemplate) -> CustomTemplate:
```

Comment:
i think the API allows creation of multiple customtemplates at once, we could do something like this
```suggestion
    def create(self, *, custom_templates: list[CustomTemplate]) -> list[CustomTemplate]:
```